subsys/emul/CMakeLists.txt

```diff
@@ -9,3 +9,4 @@ zephyr_library_sources_ifdef(CONFIG_EMUL_BMI160		emul_bmi160.c)
 
 add_subdirectory(i2c)
 add_subdirectory(espi)
+add_subdirectory(sensor)
```

Comment:
Can you clean up the bmi160 emul build handling to be consistent with this pattern?

Reply:
I actually had a thought yesterday and wanted to get your take on this (as well as @keith-zephyr). I'm thinking of moving the emulators to be in the same directory as the drivers they emulate, it would make writing and updating them much easier.